lib/client.js

```diff
@@ -128,6 +128,17 @@ class Client {
     return this._get(helper.taskPaths.deleteTask(taskId))
   }
 
+  /**
+   * Get the structure of one table.
+   * NOTE: This endpoint is not available in production yet.
```

Comment:
Please do not forget to document the function once it is in production.

Reply:
DOC added because it's already in production.